samcli/local/docker/lambda_build_container.py

```diff
@@ -163,6 +168,54 @@ def _get_container_dirs(source_dir, manifest_dir):
 
         return result
 
+    @staticmethod
+    def _convert_to_container_dirs(host_paths_to_convert, host_to_container_path_mapping):
+        """
+        Use this class to convert a list of host paths to a list of equivalent paths within the container
+        where the given host path is mounted. This is necessary when SAM CLI needs to pass path information to
+        the Lambda Builder running within the container.
+
+        If a host path is not mounted within the container, then this method simply passes the path to the result
```

Comment:
ok this is a passthrough, but searching this path within the container will not turn up anything right?

Reply:
Correct